CTestConfig.cmake.user

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file  CTestConfig.cmake.user used to for cmake ctest? I do not kown about the CTEST.